src/reaction_network.jl

```diff
@@ -391,6 +400,9 @@ function make_reaction_system(ex::Expr; name = :(gensym(:ReactionSystem)))
     species = vcat(species_declared, species_extracted)
     parameters = vcat(parameters_declared, parameters_extracted)
 
+    # Create differential expression.
+    diffexpr = create_differential_expr(options, add_default_diff, [species; parameters; variables])
+
```

Comment:
Creates an expression with all the differentials that should be declared as part of the DSL, e.g.
```julia
begin
    D = Differential(t)
end
```